src/UI/FactoryProduction.cpp

```diff
@@ -218,20 +218,26 @@ void FactoryProduction::update()
 
 	Window::update();
 
-	Renderer& r = Utility<Renderer>::get();
+	Renderer& renderer = Utility<Renderer>::get();
 
-	r.drawText(*FONT_BOLD, "Turns Completed:", rect().x() + constants::MARGIN * 2 + mProductGrid.width(), rect().y() + 25.0f, 255, 255, 255);
-	r.drawText(*FONT, string_format("%i of %i", mFactory->productionTurnsCompleted(), mProductCost.turnsToBuild()), rect().x() + constants::MARGIN * 2 + mProductGrid.width() + 120, rect().y() + 25.0f, 255, 255, 255);
+	const auto drawTitleText = [&renderer](NAS2D::Point<int> position, std::string title, std::string text) {
```

Comment:
Because you aren't changing their values, consider changing the `std::string` parameters to `const std::string&` or `std::string_view`.